docs/DeveloperGuide.adoc

```diff
@@ -297,15 +299,22 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 |Priority |As a ... |I want to ... |So that I can...
 |`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
 
-|`* * *` |user |add a new person |
+|`* * *` |project leader |add a project member | asssign tasks to them
+
+|`* * *` |project leader |add tasks |
 
-|`* * *` |user |delete a person |remove entries that I no longer need
+|`* * *` |project leader |change task status | ger reminder of the progress of each task
 
-|`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
+|`* * *` |project leader |sync team members schedules |Find a time slot where the maximum number of people, if not all, can attend
+|`* *` |Project leader with many members and tasks|Find out which group members are working on a particular task |Distribute workload evenly
+|`* *` |Project leader with tasks that require inventory|assign tasks inventory or materials that are needed |Download an inventory report whenever required
 
-|`* *` |user |hide <<private-contact-detail,private contact details>> by default |minimize chance of someone else seeing them by accident
+|`* *`|Project leader with frequent meetings|Create a new meeting with recommended time and desired location|View my next meeting in the home page
+
+|`*`|Project leader with a long-term project with ad-hoc members|Change members working on a task|Reassign tasks to incoming members
+
+|`*`|Project leader operating keeping track of purchases|Tag the inventory purchase to the member who bought it|Produce an accurate claims report at the end of the project
 
-|`*` |user with many persons in the address book |sort persons by name |locate a person easily
 |=======================================================================
```

Comment:
Did not really talk about the other type of users for this project. For example, project leader can assign tasks but did not talk about if the teammates can see the tasks